src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERN = 7;
+  const SHORT_TERN = 3;
+  const LONG_TERN_DISKOUNT = 50;
+  const SHORT_TERN_DISKOUNT = 20;
+
+  let price = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERN) {
+    return (price -= LONG_TERN_DISKOUNT);
```

Comment:
The use of '-=' in 'price -= LONG_TERN_DISKOUNT' is correct, but the variable name 'LONG_TERN_DISKOUNT' has a typo. It should be 'LONG_TERM_DISCOUNT'.